src/main/java/jiayou/task/Deadline.java

```diff
@@ -0,0 +1,38 @@
+package jiayou.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents deadline, a subtype of the task, which has a deadline date.
+ * @author Liu Jiayao
+ */
+public class Deadline extends Task {
+    private LocalDate byTime;
```

Comment:
Could consider using "due" or "by" to be more clear as the LocalDate variable could have only dates and no time.